r-p-s.js

```diff
@@ -0,0 +1,34 @@
+export function getRandomThrow() {
+    const randomNumber = Math.ceil(Math.random() * 3);
+    console.log(randomNumber);
+    if (randomNumber === 1) {
+        return 'rock';
+    } else if (randomNumber === 2) {
+        return 'paper';
+    } else {
+        return 'scissors';
+    }
+}
+
+export function checkResult(computer, player) {
+    if (computer === 'rock' && player === 'rock')
+        return 'draw';
+    if (computer === 'rock' && player === 'paper')
```

Comment:
Some of these results are a bit off. If I choose paper and the computer chooses rock, its a win for me.